src/Hooks/useAxios.jsx

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+import { useState } from 'react';
+
+axios.defaults.baseURL = "api/";
+
+export const useAxios = () => {
+    const [response, setResponse] = useState(null);
+    const [error, setError] = useState();
+    const [loading, setLoading] = useState(false);
+
+    const fireRequest = async (params) => {
+         setError();
```

Comment:
consider setting to either false/""/null instead of just empty ()
makes it easier to understand state is being reset